SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2121,13 +2138,90 @@ int32 USpatialNetDriver::ServerReplicateActors(float DeltaSeconds)
 	FSpatialLoadBalancingHandler MigrationHandler(this);
 	FSpatialNetDriverLoadBalancingContext LoadBalancingContext(this, ConsiderList);
 
-	bool bHandoverEnabled = USpatialStatics::IsHandoverEnabled(this);
+	TSet<Worker_EntityId_Key> EntitiesHandedOver;
```

Comment:
I wonder if this added code can be moved to a small class, living on stack here - it'd be somewhat less confusing to track these sets of actors and entities.

Reply:
Yeah, that's a bit of a prototype for now. Eventually there should be something like FSpatialLoadBalancingHandler.
But since it also handles the ReplicationGraph, I just avoided the complexity for now.

Reply:
+1 this is some really deep nesting in an already ridiculous function. If we are merging into master I think we have to make it so that this is at least slightly hidden